Source/WTF/wtf/Compiler.h

```diff
@@ -41,8 +41,16 @@
 #define COMPILER_HAS_CLANG_BUILTIN(x) 0
 #endif
 
+/* COMPILER_HAS_CLANG_ATTRIBUTE() - whether the compiler supports a particular attribute. */
+/* https://clang.llvm.org/docs/LanguageExtensions.html#has-attribute */
+#ifdef __has_attribute
+#define COMPILER_HAS_CLANG_ATTRIBUTE(x) __has_attribute(x)
```

Comment:
This should be just COMPILER_HAS_ATTRIBUTE since it doesn't have anything to do with clang here.
Also, can we use __has_cpp_attribute instead?  That's documented here: https://en.cppreference.com/w/cpp/feature_test#Attributes

Reply:
> This should be just COMPILER_HAS_ATTRIBUTE since it doesn't have anything to do with clang here.

It appears that `__has_attribute()` is clang-specific, just like `__has_feature()`.

> Also, can we use __has_cpp_attribute instead? That's documented here: https://en.cppreference.com/w/cpp/feature_test#Attributes

 No, `__has_cpp_attribute()` won't work (change the test case to see it won't work):  <https://godbolt.org/z/nrehshT3o>

Reply:
You're right __has_cpp_attribute does not work in this case.